website_multi_theme/models/website.py

```diff
@@ -120,6 +124,10 @@ def _multi_theme_activate(self):
         # Patterns that will be duplicated to enable multi themes
         assets_pattern = self.env.ref("website_multi_theme.assets_pattern")
         layout_pattern = self.env.ref("website_multi_theme.layout_pattern")
+
+        def view2module(view):
+            return view and view.model_data_id.module
```

Comment:
I'm not sure if you know that Odoo lets you browse empty recordsets, returning a falsey value for all fields.

Thus, you can use a variable instead of a method if you want, just below line 141 you can write:

```python
copied_view_module = copied_view.model_data_id.module
```

It will be `False` if `len(copied_view)==0`.

Reply:
Yes, I know. For some reason, I thought that it could be False in the middle of the chain, so I decided just to make the extra check. But it's not needed here. You are right